doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+RIOT Developer Memo Format, Publishing and Maintenance Process
+=========
+
+- RIOT Developer Memo: 0  
+- Author(s): Emmanuel Baccelli  
+- Date: December 2016
+
+## Abstract
+
+This memo describes a template for memos produced by the RIOT maintainers 
+community, targeting primarily RIOT developers.
+
+
+## Status
+
+This document is a product of the community of RIOT maintainers, and aims to 
+represent the consensus within this community.
+The content of this document is licensed with a Creative Commons CC-BY-SA 
+license.
+
+## Terminology
+
+This memo uses the following terminology and acronyms:
+
+- RIOT-MC: RIOT Maintainers Community
+- RDM: RIOT Developer Memo
+- PR: git pull-request to RIOT's master branch on GitHub 
```

Comment:
Nit: `pull request` is not a terminology from git, but (in our case) from GitHub (also for BitBucket, but GitLab calls them Merge Request, btw.). The correct git lingua would probably be [request-pull](https://git-scm.com/docs/git-request-pull), but that doesn't sound cool. So what about

PR: ~~git~~ **P**ull-**R**equest to RIOT's master branch on GitHub
(without bold face in the result text)